src/mrpro/operators/FourierOp.py

```diff
@@ -223,3 +224,23 @@ def adjoint(self, x: torch.Tensor) -> tuple[torch.Tensor,]:
             x = x.permute(*unpermute)
 
         return (x,)
+
+    def __repr__(self):
+        """Representation method for Fourier Operator."""
+        if self._nufft_dims:
+            string = (
+                f'Dimension along which NUFFT is applied: {self._nufft_dims}\n'
+                f'Dimension which is ignored in NUFFT: {self._ignore_dims!s}\n'
```

Comment:
ignore_dims is neither NUFFT nor FFT
```suggestion
                f'Dimension(s) which are ignored: {self._ignore_dims!s}\n'
```

Reply:
Maybe consider to put this either at the very top or the end of the string. no need to repeat it in FFT